src/index.html

```diff
@@ -0,0 +1,151 @@
+      content="https://interns2024.pages.dev/assets/og/game-home.webp"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/brands.min.css"
+      integrity="sha512-8RxmFOVaKQe/xtg6lbscU9DU0IRhURWEuiI0tXevv+lXbAHfkpamD4VKFQRto9WgfOJDwOZ74c/s9Yesv3VvIQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <script
+      src="https://kit.fontawesome.com/ebfd4f4467.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Take interns to HackerSpace</title>
+  </head>
+  <body>
+    <section class="ending">
+      <nav>
+        <ul>
+          <li>
+            <a href="https://hackerrank.com/interns/2024">Interns 2024</a>
+          </li>
+          <li><a href="https://hackerrank.com/interns/2024">Go Back</a></li>
+        </ul>
+      </nav>
+      <main>
+        <video class="win" autoplay loop muted>
+          <source
+            src="https://interns2024.pages.dev/assets/game/videos/win.mp4"
+          />
+        </video>
+        <video class="lose" autoplay loop muted>
+          <source
+            src="https://interns2024.pages.dev/assets/game/videos/lose.mp4"
+          />
+        </video>
+        <div class="button-group">
+          <button onclick="window.location.reload()">Try again</button>
+          <button class="socials" id="socials">
+            <p>Share on socials</p>
+            <div class="social-links">
+              <a
+                href="https://twitter.com/intent/tweet?url=hackerrank.com/interns/2024/game&text="
+                target="_blank"
+                title="twitter"
+                id="twitter"
+              >
+                <i class="fa-brands fa-twitter"></i>
+              </a>
+              <a
+                href="https://www.linkedin.com/shareArticle?mini=true&url=hackerrank.com/interns/2024/game&title=Help%20Hari%20take%20hackerrankers%20to%20HackerSpace"
+                title="linkedin"
+                target="_blank"
+                id="linkedin"
+              >
+                <i class="fa-brands fa-linkedin"></i>
+              </a>
+              <a
+                href="https://www.facebook.com/sharer/sharer.php?u=hackerrank.com/interns/2024/game"
+                target="_blank"
+                title="facebook"
+                i="facebook"
+              >
+                <i class="fa-brands fa-facebook"></i>
+              </a>
+            </div>
+          </button>
+        </div>
+      </main>
+    </section>
+
+    <section class="mobile">
+      <nav><h1>Interns 2024</h1></nav>
+      <div class="dialogue">
+        <img
+          src="https://interns2024.pages.dev/assets/game/sprites/spaceship.png"
+        />
+        <h3>Uh oh!</h3>
+        <p>
+          We don't support game on smaller devices. Please open on larger
+          devices.
+        </p>
+      </div>
+    </section>
+    <script src="https://unpkg.com/kaboom@3000.0.1/dist/kaboom.js"></script>
```

Comment:
**suggestion:** When you're uploading assets to hrcdn. Can you host this also? Not a big deal, but I'm just concerned what if this gets purged years down the lane.